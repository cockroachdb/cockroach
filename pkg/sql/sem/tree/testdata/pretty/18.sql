WITH t(x) AS (WITH t(x) AS (SELECT 1) SELECT x * 10 FROM t) SELECT x + 2 FROM t
