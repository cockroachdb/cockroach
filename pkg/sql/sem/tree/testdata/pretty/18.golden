1:
-
WITH
	t
		(
			x
		)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				*
				10
			FROM
				t
		)
SELECT
	x
	+
	2
FROM
	t

6:
------
WITH
	t
		(
			x
		)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				*
				10
			FROM
				t
		)
SELECT
	x
	+
	2
FROM t

7:
-------
WITH
	t
		(
			x
		)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				*
				10
			FROM
				t
		)
SELECT
	x
	+ 2
FROM t

9:
---------
WITH
	t (x)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				*
				10
			FROM
				t
		)
SELECT
	x + 2
FROM t

12:
------------
WITH
	t (x)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				*
				10
			FROM
				t
		)
SELECT x + 2
FROM t

18:
------------------
WITH
	t (x)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				*
				10
			FROM t
		)
SELECT x + 2
FROM t

19:
-------------------
WITH
	t (x)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				*
				10
			FROM t
		)
SELECT x + 2 FROM t

20:
--------------------
WITH
	t (x)
		AS (
			WITH
				t
					(
						x
					)
					AS (
						SELECT
							1
					)
			SELECT
				x
				* 10
			FROM t
		)
SELECT x + 2 FROM t

21:
---------------------
WITH
	t (x)
		AS (
			WITH
				t (x)
					AS (
						SELECT
							1
					)
			SELECT
				x
				* 10
			FROM t
		)
SELECT x + 2 FROM t

22:
----------------------
WITH
	t (x)
		AS (
			WITH
				t (x)
					AS (
						SELECT
							1
					)
			SELECT
				x * 10
			FROM t
		)
SELECT x + 2 FROM t

25:
-------------------------
WITH
	t (x)
		AS (
			WITH
				t (x)
					AS (
						SELECT
							1
					)
			SELECT x * 10
			FROM t
		)
SELECT x + 2 FROM t

32:
--------------------------------
WITH
	t (x)
		AS (
			WITH
				t (x)
					AS (
						SELECT 1
					)
			SELECT x * 10 FROM t
		)
SELECT x + 2 FROM t

33:
---------------------------------
WITH
	t (x)
		AS (
			WITH
				t (x)
					AS (SELECT 1)
			SELECT x * 10 FROM t
		)
SELECT x + 2 FROM t

35:
-----------------------------------
WITH
	t (x)
		AS (
			WITH
				t (x) AS (SELECT 1)
			SELECT x * 10 FROM t
		)
SELECT x + 2 FROM t

36:
------------------------------------
WITH
	t (x)
		AS (
			WITH t (x) AS (SELECT 1)
			SELECT x * 10 FROM t
		)
SELECT x + 2 FROM t

57:
---------------------------------------------------------
WITH
	t (x)
		AS (
			WITH t (x) AS (SELECT 1) SELECT x * 10 FROM t
		)
SELECT x + 2 FROM t

58:
----------------------------------------------------------
WITH
	t (x)
		AS (WITH t (x) AS (SELECT 1) SELECT x * 10 FROM t)
SELECT x + 2 FROM t

60:
------------------------------------------------------------
WITH
	t (x) AS (WITH t (x) AS (SELECT 1) SELECT x * 10 FROM t)
SELECT x + 2 FROM t

61:
-------------------------------------------------------------
WITH t (x) AS (WITH t (x) AS (SELECT 1) SELECT x * 10 FROM t)
SELECT x + 2 FROM t

81:
---------------------------------------------------------------------------------
WITH t (x) AS (WITH t (x) AS (SELECT 1) SELECT x * 10 FROM t) SELECT x + 2 FROM t
